src/pages/docs/degen/index.md

```diff
@@ -155,41 +153,26 @@ npm init
 npm install @golem-sdk/golem-js
 ```
 
+Make sure you have created an app-key and exported its valua as [`YAGNA_APPKEY``](http://localhost:3000/docs/degen#creating-a-unique-app-key).
+
 Next, create a file named requestor.mjs and paste the following content into it. This script sets up a task to execute node -v on the Golem Network and displays the result in your terminal.
 
 ```js
 import {
-  TaskExecutor,
-  ProposalFilters,
-  MarketHelpers,
-} from '@golem-sdk/golem-js'
-
-// Prepare the price filter, prices are in GLM
-const acceptablePrice = ProposalFilters.limitPriceFilter({
-  start: 1,
-  cpuPerSec: 1 / 3600,
-  envPerSec: 1 / 3600,
-})
+  TaskExecutor
+} from '@golem-sdk/golem-js';
 
-// Collect the whitelist
-const verifiedProviders = await MarketHelpers.getHealthyProvidersWhiteList()
-
-// Prepare the whitelist filter
-const whiteList = ProposalFilters.whiteListProposalIdsFilter(verifiedProviders)
-
-;(async function main() {
+(async function main() {
   const executor = await TaskExecutor.create({
     // The image you'd like to run on the provider(s)
     package: 'golem/node:latest',
 
     // How much you wish to spend
     budget: 0.5,
-    proposalFilter: async (proposal) =>
-      (await acceptablePrice(proposal)) && (await whiteList(proposal)),
 
     // Which network you want to spend GLM on
     payment: {
-      network: 'polygon',
+      network: 'goerli', driver: 'erc20',
```

Comment:
Is erc20 necessary to specify on 0.13.2?

Reply:
again, in (a very rare) case users would test a higher version, if they moved from 0.15 and then needed to revert to 0.14, they would need to use this option (with erc20next) , this can be removed.